Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN poetry install --no-dev
 
 EXPOSE 5000
 
-ENV FLASK_APP=api.py
-
-ENTRYPOINT flask run --host 0.0.0.0
-
+CMD ["gunicorn", "api:app", \
+     "--bind", "0.0.0.0:5000", \
+     "--workers", "20", \
```

Comment:
Is there any documentation as part of this spike around how we ended up at 20 workers?

Reply:
I mean I can add something somewhere but I don't have the exact timings for the experimental settings anymore, I like multithreaded workers, less workers, less parallel shells etc. But I can list what I've tried.

Reply:
Yeah that would be great 👍 Just need some rationale really